pokemongo_bot/__init__.py

```diff
@@ -77,6 +78,7 @@ def tick(self):
         self.check_session(self.position[0:2])
 
         workers = [
+            SoftBanWorker,
```

Comment:
This should probably be down the list, right before catching pokemon. Since the bot can still transfer and evolve while banned.